src/firefly/html/css/global.css

```diff
@@ -107,6 +107,11 @@ div.rootStyle img {
     cursor: pointer;
 }
 
+.button.disabled {
+    color: darkgray;
```

Comment:
in Toolbarbutton.css we show disabled by 
```css
opacity: 0.3;
````
you probably do that to be consistent.